extensions/wikia/Email/templates/main.mustache

```diff
@@ -0,0 +1,96 @@
+<html>
+<head>
+	<title>Wikia</title>
+</head>
+<body>
+
+<div class="main-wrapper">
+	<table class="header-wrap">
+		<tr>
+			<td></td>
+			<td>
+				<div class="content">
+					<div class="header content-inner">
+						<a href="http://www.wikia.com" class="logo">
+							<img src="http://vignette3.wikia.nocookie.net/wikianewsletter/images/8/89/Wikia.gif/revision/latest?cb=20150330185243" alt="Wikia">
```

Comment:
As discussed, it would be preferable to use Vignette lib to do this. This is brittle and can break if vignette's internal structuring changes at some point.